R/verb-select.R

```diff
@@ -36,10 +36,7 @@ ensure_group_vars <- function(loc, data, notify = TRUE) {
   if (length(missing) > 0) {
     vars <- names(data)[missing]
     if (notify) {
-      inform(glue(
-        "Adding missing grouping variables: ",
-        paste0("`", names(data)[missing], "`", collapse = ", ")
-      ))
+      message(cli::format_message("Adding missing grouping variables: {.var {vars}}"))
```

Comment:
```suggestion
      cli::cli_inform("Adding missing grouping variables: {.var {vars}}")
```

Reply:
Now I remember why I used `message()` instead of `inform()`: one test fails under Mac OS because somehow the [snapshot changes](https://github.com/tidyverse/dbplyr/runs/6285015574?check_suite_focus=true)

Reply:
Just needed to bump the cli dependency 😄